docs/DeveloperGuide.adoc

```diff
@@ -203,14 +336,25 @@ The `Storage` component,
 * can save `UserPref` objects in json format and read it back.
 * can save the TA-Tracker data in json format and read it back.
 
+==== Saved Data
+The following diagram is a breakdown of the data managed by the Storage component.
+
+.Structure of the data stored by TA-Tracker
+image::StorageClassDiagram2.png[]
+
 `TA-Tracker` saves the following data:
 
-* a list of `Module` objects representing the modules that the user is assisting.
-** each `Module` contains a list of `Session`, representing the sessions that
-the user has completed for that module.
-** each `Module` contains a list of `Group`, representing the groups that the user is
-in charge of, such as a tutorial or lab.
-** each `Group` contains a list of `Student`, representing the students enrolled in
+* a list of `Module` objects representing the modules that the user is assisting. +
++
+Within each module, there is:
+
+** a list of `Session`, representing the `done sessions` that the user has completed for that module.
```

Comment:
does it not save the un-done sessions under Session view?

Reply:
It does! There are two lists! One for modules, and one for undone sessions